RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -151,9 +157,17 @@ namespace mkfit {
         y[ts] = tk.y();
         z[ts] = tk.z();
         d0[ts] = tk.d0BeamSpot(bspot.x, bspot.y);
+
+        phi_eta_binnor.register_entry_safe(
+            oldPhi[ts],
+            eta[ts]);  // If one is sure values are *within* axis ranges: b.register_entry(oldPhi[ts], eta[ts]);
```

Comment:
please move the comment out of line (e.g. above the code) so that the code-check doesn't mangle the code part